config/crds/bases/application-networking.k8s.aws_accesslogpolicies.yaml

```diff
@@ -0,0 +1,93 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.13.0
+  name: accesslogpolicies.application-networking.k8s.aws
+spec:
+  group: application-networking.k8s.aws
+  names:
+    categories:
+    - gateway-api
+    kind: AccessLogPolicy
+    listKind: AccessLogPolicyList
+    plural: accesslogpolicies
+    shortNames:
+    - tgp
```

Comment:
need to fix this, btw this field is used for `kubectl get tgp`, etc as a shorthand

Reply:
Fixed next commit, thanks for catching this!